website/source/docs/providers/bitbucket/r/repository.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "bitbucket"
+page_title: "Bitbucket: bitbucket_repository"
+sidebar_current: "docs-bitbucket-resource-repository"
+description: |-
+  Provides a Bitbucket Repository
+---
+
+# bitbucket\_repository
+
+Provides a Bitbucket repository resource.
+
+This resource allows you manage your repositories such as scm type, if it is
+private, how to fork the repository and other options.
+
+## Example Usage
+
+```
+# Manage your respository
+resource "bitbucket_repository" "infastracture" {
+    owner = "myteam"
+    name = "terraform-code"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `owner` - (Required) The owner of this repository. Can be you or any team you
+  have write access to.
+* `name` - (Optional) The name of the repository.
+* `scm` - (Optional) What SCM you want to use. Valid options are hg or git.
+  Defaults to git.
+* `is_private` - (Optional) If this should be private or not. Defaults to true.
+* `website` - (Optional) What the website is for this repository.
+* `has_issues` - (Optional) If this should have issues turned on or not.
```

Comment:
Cool. If it's `true` for some, but `false` for others the documentation would benefit from specifying it. Like you did for `is_private`.